gitfs/worker/sync.py

```diff
@@ -166,11 +166,19 @@ def commit(self, jobs):
             message = jobs[0]['params']['message']
         else:
             updates = set([])
+            number_of_removal = 0
+            number_of_additions = 0
             for job in jobs:
-                updates = updates | set(job['params']['add'])
-                updates = updates | set(job['params']['remove'])
-
-            message = "Update {} items".format(len(updates))
+                removal_set = set(job['params']['remove'])
+                addition_set = set(job['params']['add'])
+                number_of_removal += len(removal_set)
+                number_of_additions += len(addition_set)
+                updates = updates | removal_set | addition_set
+
+            message = "Update {} items. Added {} items. " \
```

Comment:
Can you check if there are more than 0 items? It's somehow pointless to display `Removed 0 items`.